timesketch/frontend-ng/src/components/LeftPanel/Stories.vue

```diff
@@ -24,7 +24,15 @@ limitations under the License.
       :class="$vuetify.theme.dark ? 'dark-hover' : 'light-hover'"
```

Comment:
**UX feedback:** Same like with the Sigma Rules. If there are no stories and the element does not have any action when clicked, it should not be rendered as clickable. Recommend to change line 20 to `:style="!(meta.stories.length) ? '' : 'cursor: pointer'"`

Reply:
Good catch! Fixed now.